src/core/index.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @guardian/source
+ */
+
+// we need to be explicit about the index file while these directories are also packages,
+// otherwise the node require algorithm uses the package.json
```

Comment:
```suggestion
// otherwise the node require algorithm uses the package.json
// TODO: we can drop the index file when we remove the package.jsons
```